ydb/docs/en/core/reference/ydbops/configuration.md

```diff
@@ -0,0 +1,97 @@
+# Configuring ydbops
+
+{% include [warning.md](_includes/warning.md) %}
+
+
+`ydbops` can be run by specifying all the necessary command line arguments on the command invocation. However, it has two features that allow to avoid repeating the commonly used arguments:
+
+- [Config file](#config-file)
+- [Environment variables](#environment-variables)
+
+## Config file {#config-file}
+
+The configuration file for `ydbops` is a YAML-formatted file containing multiple profiles. Profiles for `ydbops` work in the same way as profiles in [{{ ydb-short-name }} CLI](../ydb-cli/profile/index.md) do.
+
+Certain command line options can be written in the configuration file instead of being specified directly in the `ydbops` invocation.
+
+### Examples
+Calling the `ydbops restart` command without a profile:
+
+```bash
+ydbops restart \
+ -e grpc://<hostname>:2135 \
+ --kubeconfig ~/.kube/config \
+ --k8s-namespace <k8s-namespace> \
+ --user admin \
+ --password-file ~/<password-file> \
+ --tenant --tenant-list=my-tenant
+```
+
+Calling the same `ydbops restart` command with profile options enabled makes the command much shorter:
+
+```bash
+ydbops restart \
+ --config-file ./config.yaml \
+ --tenant --tenant-list=<tenant-name>
+```
+
+For the invocation above, the following `config.yaml` is assumed to be present:
```

Comment:
How does it relate to YDB CLI config?
To my mind YDB CLI has ~/.ydb/... files including config.yaml? Is it the same?
If it is not the same, I expect it to be in ~/.ydb directory